### ISSUE TITLE
[BUG] fail to use custom configurations for coordinator

### ISSUE THREAD

**Issue by thweetkomputer** [alibaba/GraphScope#4329]
**Describe the bug**
The coordinator's resource configuration is hard-coded. Users fail to use custom configurations.

<img width="1036" alt="image" src="https://github.com/user-attachments/assets/c6326506-1f48-422c-b72a-7a4fcc000808">

**To Reproduce**
```
import graphscope

session = graphscope.session(
			k8s_coordinator_cpu=1)
```

Then use `kubectl` to describe the coordinator pod, the CPU is still the default value(0.5)

**Expected behavior**
The CPU should be 1

**Screenshots**

**Environment (please complete the following information):**
 - GraphScope version: [0.29.0]
 - OS: Ubuntu 
 - Version 22.04
 - Kubernetes Version [v1.30.2]

**Additional context**

**Comment by siyuan0322** [comment]
Yeah I thought this coordinator shouldn't cause much user attention, so for simplicity it's hardcoded. Any scenario that needs to customize:

**Comment by thweetkomputer** [comment]
> Yeah I thought this coordinator shouldn't cause much user attention, so for simplicity it's hardcoded. Any scenario that needs to customize:

Hi Siyuan, I tried the kgs dataset on LDBC(https://ldbcouncil.org/benchmarks/graphalytics/). I used pandas to read the data and load the data through pandas, there is an OOM in the coordinator pod. I changed the memory of the coordinator to 2Gi and it's working now.
